packaging/suse/_service

```diff
@@ -0,0 +1,27 @@
+<services>
+    <service name="tar_scm" mode="disabled">
+        <param name="url">https://github.com/%%REPOSITORY%%.git</param>
+        <param name="scm">git</param>
+        <param name="revision">%%VERSION%%</param>
+        <param name="exclude">.git</param>
+        <param name="exclude">.github</param>
+        <param name="extract">web/frontend/package.json</param>
+        <param name="extract">web/frontend/package-lock.json</param>
+        <param name="exclude">web/frontend/package-lock.json</param>
+        <param name="versionformat">0.4.0+%ct.%h</param>
```

Comment:
This is still hard-coded, and this is the actual `%%VERSION%%` parameter. 😉 
It should probably be sedded as well, like this: https://github.com/SUSE/sap_host_exporter/blob/aaa32755fedd8b687dfbc6373a95dc6224cf18f6/Makefile#L74

Reply:
Correct